Makefile

```
@@ -32,6 +32,9 @@ install-runic:
 test:
 	${JULIA} --project -e 'using Pkg; Pkg.test()'
 
+search-benchmarks:
+	cd test/search && ${JULIA} --project=../.. run_benchmarks.jl
```

Not sure if this messes up any path stuff in the script (since this will change the working directory). But if it does, it should probably be changed to use `@__DIR__` anyway when it tries to find files with relative paths.

```suggestion
	${JULIA} --project test/search/run_benchmarks.jl
```